Source/ASDisplayNodeExtras.mm

```diff
@@ -65,7 +65,7 @@ ASInterfaceState ASInterfaceStateForDisplayNode(ASDisplayNode *displayNode, UIWi
         // if not already, about to be set to invisible as it is not possible for an element to be visible
         // while outside of a window.
         ASInterfaceState interfaceState = displayNode.pendingInterfaceState;
-        return (window == nil ? (interfaceState &= (~ASInterfaceStateVisible)) : interfaceState);
+        return (window == nil ? (interfaceState & (~ASInterfaceStateVisible)) : interfaceState);
```

Comment:
while Xcode may flag this, I think the comment above indicates its intentional.

Reply:
regardless, since I have no idea, but that seems more in-depth thinking is required than the other one. I think splitting this PR up is best. 

For this change, I think I'd want unit tests that ensure the existing behavior persists. I've no idea how much work that would be w/o an expert weighing in on this one.

Reply:
Opened a [new PR](https://github.com/TextureGroup/Texture/pull/1961) for this. If you still want a unit test after reading my description, I'm happy to add one :)